conf/config.toml

```diff
@@ -116,6 +116,10 @@
 ## When PD fails to receive the heartbeat from a store after the specified period of time,
 ## it adds replicas at other nodes.
 # max-store-down-time = "30m"
+## Controls the time interval between write hot regions info into leveldb
+# hot-regions-write-interval= "20m"
```

Comment:
I suggest to change the default  to:
```
hot-regions-write-interval= "10m"
hot-regions-reserved-days= "7"
```

Reply:
I prefer to use `hot-regions-reserved-time` so that we can use `h`, `m` as the unit.

Reply:
> I suggest to change the default to:
> 
> ```
> hot-regions-write-interval= "10m"
> hot-regions-reserved-days= "7"
> ```

change to code